docs/src/main/sphinx/connector/hive.rst

```diff
@@ -956,6 +956,13 @@ The following operations are not supported when ``avro_schema_url`` is set:
 Procedures
 ----------
 
+Use the :doc:`/sql/call` statement to perform data manipulation or 
```

Comment:
We should mention that the procedure needs to be qualified with the catalog name. For example, if the catalog is named `hive`, the call should be
```sql
CALL hive.system.example_procedure()